cromulent/reader.py

```diff
@@ -1,116 +1,128 @@
-from . import model
-from .model import factory, DataError, OrderedDict, BaseResource, KEY_ORDER_HASH as koh
-from .model import STR_TYPES
-from .multiple_instantiation import EoEActivity
+
+# gvp - fix for win
+import model
+from model import factory, DataError, OrderedDict, BaseResource, KEY_ORDER_HASH as koh
+from model import STR_TYPES
+from multiple_instantiation import EoEActivity
+#from . import model
```

Comment:
As above, I'm very surprised that this would not work on windows, as it has been a core part of the python import syntax for a long time!